### ISSUE TITLE
Multiclient support

### ISSUE THREAD

**Issue by paulirish** [johnspackman/chrome-remote-multiplex#5]
I believe this project may not be necessary any longer:
https://developers.google.com/web/updates/2017/10/devtools-release-notes#multi-client

Give it a whirl and file bugs if there are any inconsistencies.

**Comment by johnspackman** [comment]
Yes, I believe that you're right - I've not tried it yet because this project also controls the lifecycle of tabs, which is useful to avoid resource leaks in long running server applications

**Comment by Bnaya** [comment]
@johnspackman Thanks for that tool :)
